Lib/configparser.py

```diff
@@ -67,6 +68,10 @@
         converter gets its corresponding get*() method on the parser object and
         section proxies.
 
+        When `allow_unnamed_section` is True (default: False), options
+        without section are accepted: the section for these is
+        ``configparser.UNNAMED_SECTION``.
```

Comment:
I would propose the add the following sentence after this one: "In the file, they are placed before the first explicit section header. This also allows reading and writing files that don't contain any explicit section headers."